<!doctype html>
<!--
  Minimal Mistakes Jekyll Theme 4.21.0 by Michael Rose
  Copyright 2013-2020 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE
-->
<html lang="en" class="no-js">
  <head>
    <meta charset="utf-8">

<!-- begin _includes/seo.html --><title>Softwares and Web-tools - Fraternali Lab</title>
<meta name="description" content="Fraternali Lab @ ISMB, UCL">


  <meta name="author" content="Fraternali Lab">
  


<meta property="og:type" content="website">
<meta property="og:locale" content="en_US">
<meta property="og:site_name" content="Fraternali Lab">
<meta property="og:title" content="Softwares and Web-tools">
<meta property="og:url" content="/software/">


  <meta property="og:description" content="Fraternali Lab @ ISMB, UCL">



  <meta property="og:image" content="/assets/images/ffgroup_logo_small.png">



  <meta name="twitter:site" content="@FraternaliLab">
  <meta name="twitter:title" content="Softwares and Web-tools">
  <meta name="twitter:description" content="Fraternali Lab @ ISMB, UCL">
  <meta name="twitter:url" content="/software/">

  
    <meta name="twitter:card" content="summary">
    
      <meta name="twitter:image" content="/assets/images/ffgroup_logo_small.png">
    
  

  







  

  


<link rel="canonical" href="/software/">




<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "Person",
      "name": "Fraternali Lab",
      "url": "/"
    
  }
</script>






<!-- end _includes/seo.html -->


<link href="/feed.xml" type="application/atom+xml" rel="alternate" title="Fraternali Lab Feed">

<!-- https://t.co/dKP3o1e -->
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5/css/all.min.css">

<!--[if IE]>
  <style>
    /* old IE unsupported flexbox fixes */
    .greedy-nav .site-title {
      padding-right: 3em;
    }
    .greedy-nav button {
      position: absolute;
      top: 0;
      right: 0;
      height: 100%;
    }
  </style>
<![endif]-->



    <!-- start custom head snippets -->

<!-- insert favicons. use https://realfavicongenerator.net/ -->

<!-- end custom head snippets -->

  </head>

  <body class="layout--single">
    <nav class="skip-links">
  <h2 class="screen-reader-text">Skip links</h2>
  <ul>
    <li><a href="#site-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->

    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        
          <a class="site-logo" href="/"><img src="/assets/images/ffgroup_logo_small.png" alt=""></a>
        
        <a class="site-title" href="/">
          Fraternali Lab
          <span class="site-subtitle">Institute of Structural and Molecular Biology, University College London</span>
        </a>
        <ul class="visible-links"><li class="masthead__menu-item">
              <a href="/about/">About</a>
            </li><li class="masthead__menu-item">
              <a href="/people/">People</a>
            </li><li class="masthead__menu-item">
              <a href="/research/">Research</a>
            </li><li class="masthead__menu-item">
              <a href="/publications/">Publications</a>
            </li><li class="masthead__menu-item">
              <a href="/software/">Software and Web-tools</a>
            </li></ul>
        
        <button class="greedy-nav__toggle hidden" type="button">
          <span class="visually-hidden">Toggle menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>


    <div class="initial-content">
      



<div id="main" role="main">
  


  <article class="page" itemscope itemtype="https://schema.org/CreativeWork">
    <meta itemprop="headline" content="Softwares and Web-tools">
    
    
    

    <div class="page__inner-wrap">
      
        <header>
          <h1 id="page-title" class="page__title" itemprop="headline">Softwares and Web-tools
</h1>
          


        </header>
      

      <section class="page__content" itemprop="text">
        
        <p><strong><a href="/biomembrane-simulations/">Biomembrane simulations</a></strong></p>

<p><a href="/biomembrane-simulations"><img src="/assets/images/TOC_file.png" alt="image-center" class="align-center" /></a></p>

<p>Lipid parameters for Molecular Dynamics simulations of biomembrane have been updated to the most recent version of the GROMOS 54a8 force field to improve solvation properties in the head region. The resulting parameters and topology files can be found <a href="/biomembrane-simulations/">here</a>.</p>

<p><strong><a href="/unippin/">UNIPPIN</a></strong></p>

<p><a href="/unippin/"><img src="/assets/images/UNIPPINLogo-1.png" alt="image-center" class="align-center" /></a></p>

<p><a href="/unippin/">UNIPPIN</a> is an integrated protein-protein interaction network the group has been collecting and curating for our analyses on protein-protein interactions. Find out more <a href="/unippin/">here</a>.</p>

<p><strong><a href="http://mabra.biomed.kcl.ac.uk/BRepertoire/">BRepertoire</a></strong></p>

<p><a href="http://mabra.biomed.kcl.ac.uk/BRepertoire/"><img src="/assets/images/Brep_logo.png" alt="image-center" class="align-center" /></a></p>

<p><a href="http://mabra.biomed.kcl.ac.uk/BRepertoire/">BRepertoire</a> is a suite of user-friendly web-based software tools for large-scale statistical analyses of antibody repertoire data. It works on data preprocessed by IMGT, and performs statistical and comparative analyses on-the-fly with versatile plotting options.</p>

<p>Cite: Margreitter C, Lu HC, Townsend C, Stewart A, Dunn-Walters DK, Fraternali F <strong>BRepertoire: a user-friendly web server for analysing antibody repertoire data</strong>, <em>Nucleic Acids Research</em>, gky276, <a href="https://doi.org/10.1093/nar/gky276">https://doi.org/10.1093/nar/gky276</a></p>

<p><strong><a href="https://titindb.kcl.ac.uk/">TITINdb</a></strong></p>

<p><a href="https://titindb.kcl.ac.uk/"><img src="/assets/images/TITINdb_logo.jpg" alt="image-center" class="align-center" /></a></p>

<p><a href="https://titindb.kcl.ac.uk/">TITINdb</a> is a web application which integrates titin structure, sequence, isoform, variant and disease information. Disease-associated nonsynonymous SNPs (nsSNPs) from the literature and population nsSNPs from the 1000 genomes project and ExAC database to be visualised on structure. Additionally the impact of these nsSNPs has been predicted computationally using both structure and sequence based methods.</p>

<p>Cite: Laddach A, Gautel M, Fraternali F <strong>TITINdb: a Computational Tool to Assess Titinâ€™s Role as a Disease Gene.</strong> <em>Bioinformatics</em> 2017 btx424. doi: 10.1093/bioinformatics/btx424, PMID: [29077808](http://www.ncbi.nlm.nih.gov/pubmed/29077808</p>

<p><strong><a href="http://fraternalilab.kcl.ac.uk/PinSnps/">PinSnps</a></strong></p>

<p><a href="http://fraternalilab.kcl.ac.uk/PinSnps/"><img src="/assets/images/PinSNPS_logo.png" alt="image-center" class="align-center" /></a></p>

<p><a href="http://fraternalilab.kcl.ac.uk/PinSnps/">PinSnps</a> is an interactive tool for the analysis of common variants, disease-related single-nucleotide polymorphisms (SNPs), Post-Translational Modifications (PTMs) and related functional sites mapped onto human protein interaction networks. One can search for and retrieve information about SNPs data and interacting proteins of the query.</p>

<p>Cite: Lu, HC, Herrera Braga, J, Fraternali, F. <strong>PinSnps: structural and functional analysis of SNPs in the context of protein interaction networks.</strong> <em>Bioinformatics</em> 32(16):2534-6, 2016. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27153707">27153707</a></p>

<p><strong><a href="http://popscomp.org:3838">POPSCOMP</a></strong></p>

<p><a href="http://popscomp.org:3838"><img src="/assets/images/PopScomp.gif" alt="image-center" class="align-center" /></a></p>

<p><a href="http://popscomp.org:3838">POPSCOMP</a> is a method to analyse interactions between individual complex components of proteins and/or nucleic acids by calculating the solvent accessible surface area (SASA) buried upon complex formation. It is based on POPS, which calculates SASAs of individual proteins and nucleic acid molecules at atomic (default) and residue (coarse-grained) level.</p>

<p>The underlying heuristic algorithm of POPS uses a geometric approximation to the shielding of atomic surface areas by nearest neighbour atoms.</p>

<p>Cite:</p>

<ol>
  <li>Kleinjung, J, Fraternali, F. <strong>POPSCOMP: An automated interaction analysis of biomolecular complexes.</strong> Nuc. Acids Res. 33:W342-W346, 2005 PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15980485">15980485</a></li>
  <li>Cavallo, L, Kleinjung, J, Fraternali, F. <strong>POPS: a fast algorithm for Solvent Accessible Surface Areas at atomic and residue level.</strong> Nuc. Acids Res. 31:3364-3366, 2003 PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12824328">12824328</a></li>
</ol>

        
      </section>

      <footer class="page__meta">
        
        


        

      </footer>

      

      
    </div>

    
  </article>

  
  
</div>

    </div>

    

    <div id="footer" class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    

    
      
        
          <li><a href="https://twitter.com/FraternaliLab" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
        
      
        
      
        
          <li><a href="https://github.com/FraternaliLab" rel="nofollow noopener noreferrer"><i class="fab fa-fw fa-github" aria-hidden="true"></i> GitHub</a></li>
        
      
        
      
        
      
        
      
    

    <li><a href="/feed.xml"><i class="fas fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2023 Fraternali Lab. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    
  <script src="/assets/js/main.min.js"></script>










  </body>
</html>
